src/components/05_pages/Content/Content.js

```diff
@@ -169,7 +170,9 @@ class Content extends Component {
                   <TableCell>
                     {(status && 'Published') || 'Unpublished'}
                   </TableCell>
-                  <TableCell>{changed}</TableCell>
+                  <TableCell>
+                    {format(new Date(changed * 1000), 'MM/DD/YYYY - HH:mm')}
```

Comment:
I'm wondering why we are not using something like: ```var options = { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' };
console.log(new Intl.DateTimeFormat('de-DE', options).format(date));``` That way we don't need a library, while we also get localization based upon the browser for free.